citations-impl/impl/src/java/org/sakaiproject/citation/impl/CitationListAccessServlet.java

```diff
@@ -363,6 +364,7 @@ protected void handleViewRequest(HttpServletRequest req, HttpServletResponse res
     			String href = citation.hasPreferredUrl() ? citation.getCustomUrl(citation.getPreferredUrlId()) : citation.getOpenurl();
     			
     			out.println("\t\t<td headers=\"details\">");
+    			out.println("\t\t\t<div class=\"detailsDiv\"><div class=\"titleDiv\"><a href=\"" + Validator.escapeHtml(href) + "\"><img id=\"" + citation.getCitationProperty("isnIdentifier") + "\" src=\"\" class=\"citationThumbnail\"></a><a href=\"" + Validator.escapeHtml(href) + "\" target=\"_blank\">" + Validator.escapeHtml( (String)citation.getCitationProperty( Schema.TITLE, true ) ) + "</a>");
```

Comment:
I just mean the page looks nicer with a placeholder.

![screen shot 2015-06-10 at 17 28 58](https://cloud.githubusercontent.com/assets/5921/8088007/38de0b7e-0f96-11e5-8a54-3b2f19dd2b50.png)